src/app/clients/clients-view/client-pay-charges/client-pay-charges.component.html

```diff
@@ -0,0 +1,46 @@
+<div class="container">
+
+  <mat-card>
+
+    <form [formGroup]="transactionForm" (ngSubmit)="submit()">
+
+      <mat-card-content>
+
+        <div fxLayout="column">
+
+          <mat-form-field>
+            <mat-label> Amount </mat-label>
+            <input matInput required autofocus formControlName="amount">
+            <mat-error *ngIf="transactionForm.controls.amount.hasError('required')">
+              Amount <strong>is required</strong>
+            </mat-error>
+          </mat-form-field>
+
+        </div>
+
+        <div fxLayout="column">
+
+          <mat-form-field>
+            <mat-label> Transaction date </mat-label>
+            <input matInput [min]="minDate" [matDatepicker]="transactionDatePicker" required formControlName="transactionDate">
+            <mat-datepicker-toggle matSuffix [for]="transactionDatePicker"></mat-datepicker-toggle>
+            <mat-datepicker #transactionDatePicker></mat-datepicker>
+            <mat-error *ngIf="transactionForm.controls.transactionDate.hasError('required')">
+              Transaction date <strong>is required</strong>
+            </mat-error>
+          </mat-form-field>
+
+        </div>
+
+      </mat-card-content>
+
+      <mat-card-actions fxLayout="row" fxLayout.xs="column" fxLayoutAlign="center" fxLayoutGap="5px">
+        <button type="button" mat-raised-button [routerLink]="['../../../../../','general']">Cancel</button>
```

Comment:
After changing the path properly in the route folder, this also needs to be modified.

Reply:
Have to solve the above problem